target/scripts/helpers/error.sh

```diff
@@ -2,7 +2,7 @@
 
 function _errex
 {
-  echo -e "Error :: ${*}\nAborting." >&2
+  printf '\e[0m[  \e[91mERROR  \e[0m]  |  %s\nAborting.\n' "${*}" >&2
```

Comment:
Analog to `setup.sh` and other places, we should use variables for colors, instead of cryptic escape codes.

What about putting this:

https://github.com/docker-mailserver/docker-mailserver/blob/b730942b96aec3ab52699b5d927577f42eef0a45/setup.sh#L22-L26

globally in, e.g. `log.sh`?

Reply:
Probably a good idea! I will tend to it.

Reply:
Yes, I just did not get around to it yesterday (was interrupted in the process). I will take care of it.

Reply:
Done in 3c7483149d53e750925fe5651df3fadac1ae239e